src/utils/utils.js

```diff
@@ -836,6 +836,14 @@ export function isBlock(node) {
     return blockTagNames.includes(tagName);
 }
 
+export function isBold(node) {
+    const fontWeight = +getComputedStyle(node).fontWeight;
+    return (
+        fontWeight > 500 ||
+        fontWeight > +getComputedStyle(closestBlock(node)).fontWeight
```

Comment:
What about `<span style="font-weight: 300">a<span style="font-weight: 400">[b]</span>c</span>`? And this function won't work for text nodes, so it should be `function isBold(element){...}` or update the function to work with text nodes. Also I think a comment explaining the condition and related scenarios would be welcome.

Reply:
In this scenario it will return `false` when it shouldn't then:
```html
<p style="font-weight: 300">
  <span style="font-weight: 400">[a]</span>
</p>
```

Reply:
Uh if I'm not mistaken it will return `true` as expected: `closestElement(node)` will return the `<span>` and `closestBlock(node)` will return the `<p>`. The font weight of the `<span>` is bigger than that of the `<p>` -> `true`.

Reply:
Ah yes, I got confused because I thought you would use `closestElement` instead of `closestBlock`, but that was about making the function work with text nodes. Then if you keep `closestBlock`, see example in first comment.